release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
 
 #### New features
 
+##### Language Server
+
+- Added API docs reference support on hover
+- Added new code actions to extract anonymous records into records and to generate undefined record types
+- Introduced new code actions to generate getters and setters for class-level variables
```

Comment:
Shall we mention that this is a `source action`?  And should also mention source actions such as this and `Optimize all imports` are not shown directly under the bulb but under `source actions` option in the right click menu?

Reply:
Done